client/js/lounge.js

```diff
@@ -181,6 +181,13 @@ $(function() {
 		}
 	});
 
+	socket.on("open", function(id) {
+		// Another client opened the channel, clear the unread counter
+		sidebar.find("[data-id='" + id + "'] .badge")
+			.removeClass("highlight")
+			.empty();
+	});
+
```

Comment:
I mean that this clears unread counter for channels opened in other clients, do we need to move the unread **marker** for these channels as well?